content/soda-gettingstarted/multicloud-ha-installation-using-ansible.md

```diff
@@ -0,0 +1,74 @@
+weight: 50
+---
+
+This document describes how to install SODA Multi Cloud project on a pre-configured Kubernetes cluster. These steps will help you to install / uninstall Multi-Cloud project in HA mode. After installation using these steps, SODA Multi-Cloud Services will be available in HA mode. You can test either through SODA Dashboard UI.
+For installing SODA in default standalone configuration please refer to the [standalone installation guide](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)
+
+## Pre-config for Deploying Multi-Cloud Services on Kubernetes Cluster (Ubuntu 16.04 or Ubuntu 18.04)
+All the installation work is tested on `Ubuntu 16.04` and `Ubuntu 18.04`, please make sure you have installed the right one. Also `root` user is REQUIRED before the installation work starts.
+The following configuations are required on the Kubernetes Cluster and the SODA installer node. 
+  1. A Kubernetes Cluster should be available for deploying the SODA Multi-Cloud Services on it.
+  2. The Kubernetes Cluster should have atleast three nodes to deploy the services. 
+  3. The Kubernetes Master node should have a user, (say 'k8s_user') configured to access the installer node.
+  4. [Passwordless SSH](https://help.ubuntu.com/community/SSH/OpenSSH/Keys) should be setup between the Kubernetes Master node and the SODA Installer node.
+
+
+All the other configurations remain same as the [standalone installation](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/). 
+Please follow the standalone installation guide for the remaining installation steps. 
+Here are the steps that are specific to the HA installation of Multi-Cloud Services on Kubernetes Cluster.
+
+
+### Configurations for Installing Multi-Cloud in HA mode.     
+The basic configurations for installing SODA Multi-Cloud in HA mode are the same as installing Multi-Cloud in the standalone mode. The few additional steps required to install Multi-Cloud in HA mode are 
+
+As a root user
+#### Download SODA installer code
+```bash
+git clone https://github.com/sodafoundation/installer.git
+cd installer/ansible
+# Checkout the required version. For example, to checkout v1.2.0 follow
+git checkout v1.2.0
+```
+{{% notice warning %}}
+Checkout the latest stable release. Current stable release: stable/faroe. If you want to get the master branch of all components, you can skip this step. (Master may not be stable or tested fully)
+{{% /notice %}}
+
+
+    - cd installer/ansible/group_vars
+    - Open the common.yml file
+      - Add the IP of the host (which is reachable from the Kubernetes node)
+      - export the HOST_IP=<IP-of-the-Host>
```

Comment:
If the user has has followed the standalone installer then this is a repeat step.

Reply:
its easy to miss , hence better to have it here as well.